scalafmt-dynamic/src/test/scala/tests/DynamicSuite.scala

```diff
@@ -413,7 +413,7 @@ class DynamicSuite extends AnyFunSuite with DiffAssertions {
     f.assertFormat()
     val configOpt = f.dynamic.configsCache
       .getFromCache(f.config)
-      .flatMap(_.toOption.map(_._1))
+      .flatMap(_.fold(_ => None, Some(_)).map(_._1))
```

Comment:
is this different?

Reply:
It's the same things but Scala 2.11's `Either` hasn't `toOption` method

Reply:
what about slightly simpler

```
val configOpt = f.dynamic.configsCache
      .getFromCache(f.config)
      .flatMap(_.fold(_ => None, Some(_._1)))
```

or even
```
val configOpt = f.dynamic.configsCache.getFromCache(f.config).collect {
  case Right((x, _)) => x
}
```

Reply:
Yes, it looks much better